src/pkg/packaging/deb/package.targets

```diff
@@ -17,14 +17,7 @@
     </PropertyGroup>
 
     <Exec Command="$(DotnetRestoreCommand) $(dotnetDebToolSource)"/>
-    <Exec Command="$(DotnetToolCommand) pack $(dotnetDebToolSource) --output $(PackagesOutDir)intermediate/ $(VersionSuffixArg)"/>
-
-    <ItemGroup>
-      <pkFiles Include="$(PackagesOutDir)intermediate/dotnet-deb-tool.*.nupkg"
-               Exclude="$(PackagesOutDir)intermediate/*.symbols.nupkg" />
-    </ItemGroup>
-
```

Comment:
Is this copy not needed any longer?

Reply:
No, it wasn’t necessary to begin with. What this was doing before was packing the nupkg to an intermediate folder and then copying the nupkg to the output folder. I just made the pack output straight to the final destination. As for the symbol package, this doesn’t get produced by default from my testing. And if it did, we can just shut it off, instead of making it and not copying it.